tests/test_dwa_stuck_case.py

```diff
@@ -0,0 +1,45 @@
+import os
```

Comment:
Could you please just add a test function in test_dynamic_window_approach.py, not adding a new file?